index.html

```diff
@@ -117,14 +121,21 @@
       	$("#updated").empty();
       	for (r=0; r < Math.min(4, updated.length); r++)
       	{
-      		repo = updated[r];
+          repo = updated[r];
           desc = repoDescription(repo);
+            
+        
           var $uitem = $("<div>").addClass("updated-card col-sm-5 col-md-4 col-lg-3");
           var $item = $("<div>").addClass("card pin " + (repo.language || '').toLowerCase() + " "+repo.name.toLowerCase());
           var $link = $("<a>").attr("href", repoUrl(repo)).appendTo($item);
           $link.append($("<h4>").text(repo.name));
           $link.append($("<h5>").text((repo.language != null) ? repo.language : ""));
-          $item.append($("<p>").text(desc));
+          if(repo.description) {
+              $item.append($("<p>").text(desc));
+          }
+          else {           
+              $item.append($("<p>").text('No description given.'));
```

Comment:
I was thinking of just not showing anything here instead, thoughts on that?

Reply:
Sure.